constants/events.ts

```diff
@@ -39,6 +44,7 @@ export const ShowEvents: Event[] = [
         "artist",
       ],
     ],
+    // the ticket DMs will be sent by this pubkeys, and the mobile app will only listen for DMs from this pubkey
     pubkey: "7759fb0b22fa80cc48ba83c4ce0c90d86ac4e7ae7fad4a847ead2cf5d27031af",
```

Comment:
Update to new ticketbot pubkey
```suggestion
    pubkey: "7759fb14ce9324de4c450d42dbbbd011d3540b49ce91a76912a27738f9a97be4",
```